mne_bids/tests/test_read.py

```diff
@@ -332,6 +353,12 @@ def test_get_head_mri_trans_ctf():
     assert_almost_equal(trans['trans'], estimated_trans['trans'])
 
 
+@pytest.mark.filterwarnings("ignore:Input info has 'meas_date' set to None."
+                            ":RuntimeWarning:mne")
+@pytest.mark.filterwarnings(r'ignore:The unit for channel\(s\) STI 001, '
+                            'STI 002, STI 003, STI 004, STI 005, STI 006, '
+                            'STI 014, STI 015, STI 016 has changed from V to '
+                            'NA.:RuntimeWarning:mne')
```

Comment:
@pytest.mark.filterwarnings("ignore:Input info has 'meas_date' set to None."
                            ":RuntimeWarning:mne")
@pytest.mark.filterwarnings(r'ignore:The unit for channe*:RuntimeWarning:mne')